CONTRIBUTORS.md

```diff
@@ -20,18 +19,15 @@ Contributors to the Chapel 1.28.0 release
 * David Iten, [HPE]
 * Bhavani Jayakumaran, [HPE]
 * Engin Kayraklioglu, [HPE]
-* Shreyas Khandekar, [HPE] intern from [University of Arizona]
 * Vassily Litvinov, [HPE]
 * David Longnecker, [HPE]
 * Ben McDonald, [HPE] (former intern from [Gonzaga University])
-* Brandon Neth, [HPE] intern from [University of Arizona]
-* Nikki Rad, [HPE] intern from [Texas A&M University]
 * Ahmad Rezaii, [HPE]
 * Anna Rift, [HPE]
-* Thomas Rolinger, individual contributor
 * Elliot Ronaghan, [HPE]
 * Andy Stone, [HPE] (former [Cray Inc.] intern from [Colorado State University])
```

Comment:
@bradcray , this seems to imply that Andy (and Ben McD above) made the contributions while they were interns?  It struck me as odd.  I am not sure why we are mentioning they used to be an intern here?

Reply:
The intention of the wording is to nod to their past contributions and affiliations as interns on top of their present ones as HPE developers.  For devs in this category, I've given them the option to nod to their roots or shed them, where different people have gone different ways (Engin retired his in the past year or so I think, Andy and BenMcD preferred keeping theirs last I asked).

Reply:
(closing this tab at the end of the day:  feel free to offer a suggested wording that would make it clearer.  I suspect the reason I used this terse wording was to stay as close to 80 columns as possible).